templates/taxi/car_delete_confimation.html

```diff
@@ -0,0 +1,14 @@
+{% extends "base.html" %}
+{%  block content %}
+  <h2>Delete this car</h2>
+  <p>
+    Are you sure you really want to delete the information of this car?
+    ({{ car.model }})
+    <br>
+    <i>All boundaries with drivers and manufacturers will be lost</i>
+  </p>
+  <form action="" method="post">
+  {%  csrf_token %}
+  <input class="btn btn-danger" type="submit" value="Yes, delete">

```

Comment:
it will be a good option for the user to cancel deletion , that’s why we have an extra step before deleting something

Reply:
> it will be a good option for the user to cancel deletion , that’s why we have an extra step before deleting something

![image](https://github.com/mate-academy/py-taxi-service-forms/assets/140715010/6721641d-30bb-465b-8a30-ac1b41c3de0f)
I have the cancel button all over the programm